src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -166,20 +209,92 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
   readonly RoleTypes = RoleTypes
   readonly IncorporatorTypes = IncorporatorTypes
   readonly NumWord = NumWord
+  readonly Rules = Rules
 
+  /** The list of organizations/persons. */
   private get orgPersonList (): OrgPersonIF[] {
     return this.getAddPeopleAndRoleStep.orgPeople
   }
 
-  private get directorVerbiage (): string {
-    return this.getMinimumDirectorCount > 1 ? 'Directors' : 'Director'
+  /** The list of completing parties. */
+  private get completingParties (): OrgPersonIF[] {
+    return this.orgPersonList.filter(
+      people => people.roles.some(party => party.roleType === RoleTypes.COMPLETING_PARTY)
+    )
+  }
+
+  /** The list of incorporators. */
+  private get incorporators (): OrgPersonIF[] {
+    return this.orgPersonList.filter(
+      people => people.roles.some(party => party.roleType === RoleTypes.INCORPORATOR)
+    )
+  }
+
+  /** The list of directors. */
+  private get directors (): OrgPersonIF[] {
+    return this.orgPersonList.filter(
+      people => people.roles.some(party => party.roleType === RoleTypes.DIRECTOR)
+    )
+  }
+
+  /** The list of people without roles. */
+  private get peopleWithNoRoles (): OrgPersonIF[] {
+    return this.orgPersonList.filter(people => people.roles.length === 0)
+  }
+
+  /** Whether the Completing Party rule is valid. Always true if rule doesn't exist. */
+  private get validNumCompletingParty (): boolean {
+    const rule = this.getPeopleAndRolesResource.rules.find(r => r.rule === Rules.NUM_COMPLETING_PARTY)
+    if (!rule) return true
+    return rule.test(this.completingParties.length)
+  }
+
+  /** Whether the Minimum Incorporators rule is valid. Always true if rule doesn't exist. */
+  private get validMinimumIncorporators (): boolean {
+    const rule = this.getPeopleAndRolesResource.rules.find(r => r.rule === Rules.NUM_INCORPORATORS)
+    if (!rule) return true
+    return rule.test(this.incorporators.length)
```

Comment:
Here is where I evaluate this particular rule... See also my comments in the resource files.